docker/app/migrate_and_serve.sh

```diff
@@ -9,5 +9,7 @@
 
 echo [i] Applying migrations.
 python manage.py migrate --verbosity 0
+echo [i] Collecting static files.
+python manage.py collectstatic --no-input --clear --verbosity 0
```

Comment:
Same as with the Dockerfile above, but since the site is served by the Django devserver here anyways, this doesn't really have any use

Reply:
since `django-simple-bulma` depends on `collectstatic` to do its thing, it absolutely is necessary to do it locally, in the CI, and in prod.

Reply:
It does bear thinking about a little either way though, since to load the `settings.py` we need to have a few env vars set, which deliberately aren't set in the `Dockerfile`s.

I'll see if I can't just provide them in the `RUN` call like you normally would in your terminal.